src/model/learn.py

```diff
@@ -54,7 +59,7 @@ def learn():
             model.load(model_load_path, env=env)
             print("Done loading model")
 
-        model.learn(total_timesteps=500000, progress_bar=True)
+        model.learn(total_timesteps=math.ceil((simulation_duration_seconds * 1000) / simulation_step_duration_msec), progress_bar=True)
```

Comment:
This isn’t that many time steps, if you changed this to make the runtime proportional to duration and step size then you should probably add a factor so it learns for more times steps because this would result in about 1000 steps which is tiny